### PR TITLE
Working into the contact form and changing README file

### PR DESCRIPTION
**Pull request by nimplay**
Summary:

We create tha contact form and add new information about mu partner in README file.

### CODE REVIEW

**Review by cwaku — COMMENTED** [review]
Hi @nimplay

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that I cannot perform a thorough review on the PR. I only see changes to README and not the other files. Kindly create a new PR and push the commits again. To prevent this in the future, please always confirm that changes are visible when you click on "Files changes"

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by nimplay** [comment]
Summary:
last week forward the contact form for that reason did not have many changes to make. Although I did notice that I had not added the "inter" font nor had I set the character limit in the name box and had not activated the requirement for the form.

File: index.html [hunk 1]
```diff
@@ -267,35 +277,55 @@ <h3 class="ctitletext">
...
+        <input
+          type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Full name"
+          class="cdata"
+          required
+        />
         <input
           type="text"
```
Comment (m05e5) [inline index.html]: - [ ] Great job but i recommend using `type="email"` for the email field

File: index.html [hunk 2]
```diff
@@ -267,35 +277,55 @@ <h3 class="ctitletext">
         </h3>
       </div>
       <form class="contactform">
```
Comment (m05e5) [inline index.html]: - [ ] as part of the requirements, your form should uses the action and method attribute to integrate with [formspree service](https://formspree.io/html)

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @nimplay , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] _**Please check the comments below**_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NATASHA-ct — COMMENTED** [review]
- [ ] Nothing to say, you did well . Great Job.

File: index.html [hunk 3]
```diff
@@ -266,36 +276,58 @@ <h3 class="ctitletext">
...
+          required
         />
         <textarea
           name="message"
           placeholder="Enter Text Here"
           class="cmessage"
+          maxlength="500"
+          required
         ></textarea>
         <input class="csubmit" type="submit" value="Get in touch" />
```
Comment (koachcode) [inline index.html:304]: - Please add button interaction as it is shown in the Figma, such as hover, active (when it's pressed), disabled and enable (default state)

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @nimplay , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- Linters passed
- Clean code :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koachcode — APPROVED** [review]
Hi @nimplay , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**